.pipelines/release.yml

```diff
@@ -312,7 +332,7 @@ jobs:
       msbuildArgs: /p:CIBuild=true /target:PowerToysInstaller /bl:$(Build.SourcesDirectory)\msbuild.binlog
       platform: $(BuildPlatform)
       configuration: $(BuildConfiguration)
-      clean: true
+      clean: false # don't undo our hard work above by deleting the CustomActions dll
```

Comment:
this has bitten me before -- sometimes VS *STILL* decides to rebuild it. Can you validate that that's not the case, or do we need to do a test signed run?

Reply:
It's not the case, I've run it through the pipelines already, and it's not rebuilt.

Reply:
Actually adding a step to verify if it's signed after extracting from the MSI ;) That should help.